.github/workflows/variables.yml

```diff
@@ -43,6 +43,9 @@ on:
       next-milestone:
         description: Next milestone (A.B.C[-N]-rc.`D+1`)
         value: ${{format('{0}.{1}{2}-rc.{3}', jobs.parse.outputs.release, jobs.parse.outputs.patch, jobs.parse.outputs.dash-name, jobs.parse.outputs.next-rc)}}
+      next-release:
+        description: Next Release (A.`B+1`)
+        value: ${{jobs.parse.outputs.next-release}}
       next-release-patch:
         description: Next Release.patch (A.B.`C+1`)
         value: ${{format('{0}.{1}', jobs.parse.outputs.release, jobs.parse.outputs.next-patch)}}
```

Comment:
[nit] Output names `next-release`, `next-release-patch` and `next-named-release-patch` don't stack against each other any more because the new `next-release` means a different thing now.

One option that comes to mind is to rename
`next-release-patch` -> `next-patch`
and `next-named-release-patch` -> `next-named-patch`.
This way, they don't refer to `next-release`.

Reply:
What do you think about this:

`next-release` -> `next-minor-release`
`next-release-patch` -> `next-patch-release`
`next-named-release-patch` -> `next-named-patch-release`

Reply:
Looks ok.
I feel tiny bit uneasy about `minor` because it alludes to a small amount of changes, but the alternative `next-y-stream-release` looks a bit ugly. Therefore, what you suggest is actually ok.

Reply:
I've addressed this in https://github.com/stackrox/test-gh-actions/pull/158